lib/app.ts

```diff
@@ -33,6 +33,7 @@ export const setupSentry = onceAsync(async () => {
 	const config = await import('./config');
 	const Sentry = await import('@sentry/node');
 	Sentry.init({
+		autoSessionTracking: false,
```

Comment:
`@sentry/node` was updated from v5 to v6, and this added property preserves the v5 behavior. Otherwise, session HTTP requests are always made to Sentry.io even if no error occurred, which seemed unnecessary to me.